examples/static/samples/files/amp-form-success.html

```diff
@@ -0,0 +1,28 @@
+<!---
+
+hideCode: true
+hidePreview: true
+
+--->
+
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Success</title>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>  
+  <link rel="canonical" href="<%host%>/amp-form-success/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body> 
+  <!--
+  ## Success page for amp-form
+
+  Success! Thanks for trying the amp-form demo.
+  -->
+  <!--
+  Back to [amp-form](/components/amp-form) sample.
```

Comment:
Does this link work?

Reply:
No, nothing is rendered on the page. Since the example is only to show, that a page reload can be triggered via the `action` attribute, we could change the redirect in backend to just `response.redirect('back')`. This will just reload the page in the same location.

Reply:
In that case we should fix this and find a better way. I'm not really happy about simply reloading the page as users might miss it (we should change the sample phrasing as well, it's not about reloading). The simplest fix is probably to change the comment into HTML with a link back to the sample.

In the future, feel free to directly fix things like this that are obviously broken.

Reply:
Alright. For now I implemented your suggestion with a link back to the example.